net.technicpack.TechnicLauncher.metainfo.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<component type="desktop">
+    <id>Syndicate</id>
```

Comment:
build fail because this is the wrong it. Even unrelated.
```suggestion
    <id>net.technicpack.TechnicLauncher</id>
```

Reply:
I must've pasted the company name in the id slot, wow I am a fool